plugins/azure_sentinel/plugin.spec.yaml

```
@@ -529,7 +766,7 @@ actions:
       resourceGroupName:
         title: Resource Group Name
         required: true
-        description: The name of the resource group within the user's subscription
+        description: The name of the resource group within the user's subscription. The name is case-insensitive
         type: string
         example: resourcegrup1
```

This value is used a few times and could be replaced.

```suggestion
        example: resourcegroup1
```